src/dom_types/values.rs

```diff
@@ -33,6 +33,35 @@ impl From<&CSSValue> for CSSValue {
     }
```

Comment:
Maybe we can delete `impl From<&CSSValue> for CSSValue {`, because it was workaround for `style!` macro and we no longer use it directly in macro because of `.to_css_value`.
=> Try to delete it and run `cargo make verify`.

Reply:
Removed, `cargo make verify` pass